### ISSUE TITLE
Move BaseAppRedirector GA logic to App container.

### ISSUE THREAD

**Issue by chuckharmston** [mozilla/testpilot#1583]
Forked from #1372. See https://github.com/mozilla/testpilot/blob/4bebfa9e1df7313f8c805dc5b9d44791c4ea1999/frontend/src/app/lib/router.js#L42

**Comment by chuckharmston** [comment]
Adding `needs:qa` because this touches our global GA `pageview` events. We want to make sure that we are sending 1 (and only 1) `pageview` event for each logical piece of navigation on the site. We should also ensure that each view is loadable by going to the url directly and by following links.

**Comment by SoftVision-CosminMuntean** [comment]
Just finalized testing this and I think covered all the Test Pilot pages. I have only seen one pageview GA event for each of the tested pages. 
However, while testing this I have found one issue that blocks the entire GA pageview events. I have already logged it so you guys can look into it (#1601).

Tested this with the latest Nightly (52.0a1 - Build id: 20161013030204) build on dev server.
